models/rhcos_versions.go

```diff
@@ -0,0 +1,45 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"strconv"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+)
+
+// RhcosVersions rhcos versions
+//
+// swagger:model rhcos-versions
+type RhcosVersions []*OsImage
```

Comment:
remame to `OsImages` ?

Reply:
Yes I think it makes sense to name everything consistently and go with "OS Image" as the base name across the board.

Reply:
sure, changed to OS_IMAGES then.